README.md

```diff
@@ -102,6 +103,7 @@ Example trigger payload:
 The following two actions are provided by the Slack pack.
 
 * ``post_message`` - Post a message to the specified channel using an incoming webhook.
+* ``post_attachment`` - Post an attachment to the specified channel using an incoming webhook.
```

Comment:
On the description above, can we rephase it to "The following actions" are provided, rather than the following two - as it's already got the wrong count, and saves someone having to change that phrase each time they do an update.